example/src/main/java/org/apache/rocketmq/example/rpc/AsyncRequestProducer.java

```diff
@@ -28,36 +28,40 @@
 public class AsyncRequestProducer {
     private static final InternalLogger log = ClientLogger.getLog();
 
+    public static final String PRODUCER_GROUP = "please_rename_unique_group_name";
+    public static final String DEFAULT_NAMESRVADDR = "127.0.0.1:9876";
+    public static final String TOPIC = "RequestTopic";
+    public static final String TAG = "Tag";
+    public static final long TTL = 3000L;
+
     public static void main(String[] args) throws MQClientException, InterruptedException {
-        String producerGroup = "please_rename_unique_group_name";
-        String topic = "RequestTopic";
-        long ttl = 3000;
 
-        DefaultMQProducer producer = new DefaultMQProducer(producerGroup);
+        DefaultMQProducer producer = new DefaultMQProducer(PRODUCER_GROUP);
+        //You need to set namesrvAddr to the address of the local namesrv
+        producer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
```

Comment:
May cause the official website quick start to set namesrv through the command line to fail.

![image](https://user-images.githubusercontent.com/14222167/169222247-c94c3a0d-2a74-4a72-9253-afdaf04e313c.png)